src/Symfony/Component/AssetMapper/Factory/MappedAssetFactory.php

```diff
@@ -25,6 +26,7 @@ class MappedAssetFactory implements MappedAssetFactoryInterface
 {
     private const PREDIGESTED_REGEX = '/-([0-9a-zA-Z]{7,128}\.digested)/';
     private const PUBLIC_DIGEST_LENGTH = 7;
+    private const INTEGRITY_HASH_ALGORITHMS = ['sha256', 'sha384', 'sha512'];
```

Comment:
Do we really need multiple algorithms ?

Reply:
I think so: people apparently don’t agree on the default value, and it’s required to support https://w3c.github.io/webappsec-subresource-integrity/#agility